tests/Benchmarks/System.Buffers/ToSpan.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Buffers;
+using System.Buffers.Tests;
+using System.Collections.Generic;
+using System.Collections.Sequences;
+using System.Linq;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+
+namespace Benchmarks.System.Buffers
```

Comment:
Can you please share the comparison of how much memory is allocated/used up for each of those benchmarks, and not just runtime.

You can get that by using the `MemoryDiagnoser` attribute. For example:
https://github.com/dotnet/corefxlab/blob/00da38200bf18b875b013343dadf6b8566385b96/tests/Benchmarks/Microsoft.Experimental.Collections/DictionarySlimRemove.cs#L12-L13